### PR TITLE
Moves transmission rate from devices to AGCO glossary

### PR DESCRIPTION
**Pull request by waldemarnt**
We created a glossary to explain the complex domain terms
I just moved transmission rate explanation to this glossary as
was discussed with the team.

after:
screen shot 2016-08-16 at 3 43 34 pm

### CODE REVIEW

File: source/includes/glossary/_transmission_rate.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#AGCO Glossary
+
+##Transmission rate
+
+The [Telemetry API](#telemetry-api) is supplied by `Topcon` and ` AGCO Connectivity Module (ACM)` devices.
```
Comment (dclucas) [inline source/includes/glossary/_transmission_rate.md]: How about "The Telemetry API received data from (...)"?

File: source/includes/glossary/_transmission_rate.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+`AGCO` service's offers [data plans](#data-plans) that are related to how often the data is collected by the `Device` on the machine
```
Comment (dclucas) [inline source/includes/glossary/_transmission_rate.md]: Suggestion: "Fuse data plan subscriptions determine how often (...)"

File: source/includes/glossary/_transmission_rate.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+and how often this data is sent to the `Telemetry API`.
+
+### Data harvesting
+
+The data harvesting is separated in two steps:
+
+1. In the first step the `CAN variables data` are collected through the machine in an interval of time.
```
Comment (dclucas) [inline source/includes/glossary/_transmission_rate.md]: "the `CAN variables data` are collected" ==> "`CAN variable data` is collected"

File: source/includes/glossary/_transmission_rate.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+This inverval of time depends on which plan the `Equipment` belongs to. This step is called `Data Logging Rate`.
+
```
Comment (dclucas) [inline source/includes/glossary/_transmission_rate.md]: "the Equipment belongs to" => "the Equipment is assigned to"

**Comment by lucaslago** [comment]
👍

**Pull request merged**